src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Garnet;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Garnet resources to the application model.
+/// </summary>
+public static class GarnetBuilderExtensions
+{
+    /// <summary>
+    /// Adds a Garnet container to the application model.
+    /// </summary>
+    /// <remarks>
+    /// The default image is "garnet" and the tag is "1.0.2".
+    /// </remarks>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <param name="port">The host port to bind the underlying container to.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
```

Comment:
Could you add in a super concise `<example>...</example>` section. We are now starting to consciously improve our docs (the rest of the code base needs to catch up be we want to stop the bleeding).

The example should include the `CreateBuilder(args)` call and how to use the AddGarnet API. Half a dozen lines should do it.

Reply:
Improving documentation is important. I wish you success!
Added, I hope this example works.